.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.unitTest.pyTestEnabled": true,
+    "python.testing.pyTestEnabled": true,
```

Comment:
Intensional?

Reply:
yup..  
I can take it in a separate PR but thought of combining it :)

This is introduced due to some latest change in VSCode